### ISSUE TITLE
Maya usd exporter crashes on exporting instance

### ISSUE THREAD

**Issue by chloe5608** [PixarAnimationStudios/USD#275]
### Description of Issue
When exporting a Maya scene with instances to usda, by selecting export all -> export instances, Maya crashes. 
### Steps to Reproduce
1. Load attached carInstances.ma into Maya ([carInstances.ma.zip](https://github.com/PixarAnimationStudios/USD/files/1278684/carInstances.ma.zip))
2. Select File->Export All, check export instances and make references instanceable
3. Crash

### System Information (OS, Hardware)
Mac OS 10.13 (High Sierra), Mac Pro
### Package Versions
Maya 2017
### Build Flags
USD 0.8

**Comment by chloe5608** [comment]
Only check export instances is enough to trigger the crash.

**Comment by jtran56** [comment]
Filed as internal issue #150465.

**Comment by meshula** [comment]
FWIW, the crash log says: "Used null prim".

````
>	libusd.dll!pxrInternal_v0_8__pxrReserved__::Usd_IssueFatalPrimAccessError(const pxrInternal_v0_8__pxrReserved__::Usd_PrimData * p) Line 224	C++
 	libusd.dll!pxrInternal_v0_8__pxrReserved__::UsdAttribute::_CreateSpec(const pxrInternal_v0_8__pxrReserved__::SdfValueTypeName & typeName, bool custom, const pxrInternal_v0_8__pxrReserved__::SdfVariability & variability) Line 274	C++
 	libusd.dll!pxrInternal_v0_8__pxrReserved__::UsdAttribute::_Create(const pxrInternal_v0_8__pxrReserved__::SdfValueTypeName & typeName, bool custom, const pxrInternal_v0_8__pxrReserved__::SdfVariability & variability) Line 314	C++
 	libusd.dll!pxrInternal_v0_8__pxrReserved__::UsdPrim::CreateAttribute(const pxrInternal_v0_8__pxrReserved__::TfToken & name, const pxrInternal_v0_8__pxrReserved__::SdfValueTypeName & typeName, bool custom, pxrInternal_v0_8__pxrReserved__::SdfVariability variability) Line 306	C++
 	libusdGeom.dll!pxrInternal_v0_8__pxrReserved__::UsdGeomXformOp::UsdGeomXformOp(const pxrInternal_v0_8__pxrReserved__::UsdPrim & prim, const pxrInternal_v0_8__pxrReserved__::UsdGeomXformOp::Type opType, const pxrInternal_v0_8__pxrReserved__::UsdGeomXformOp::Precision precision, const pxrInternal_v0_8__pxrReserved__::TfToken & opSuffix, bool isInverseOp) Line 356	C++
 	libusdGeom.dll!pxrInternal_v0_8__pxrReserved__::UsdGeomXformable::AddXformOp(const pxrInternal_v0_8__pxrReserved__::UsdGeomXformOp::Type opType, const pxrInternal_v0_8__pxrReserved__::UsdGeomXformOp::Precision precision, const pxrInternal_v0_8__pxrReserved__::TfToken & opSuffix, bool isInverseOp) Line 216	C++
 	libusdMaya.dll!pxrInternal_v0_8__pxrReserved__::MayaTransformWriter::pushTransformStack(const MFnTransform & iTrans, const pxrInternal_v0_8__pxrReserved__::UsdGeomXformable & usdXformable, bool writeAnim) Line 404	C++
 	libusdMaya.dll!pxrInternal_v0_8__pxrReserved__::MayaTransformWriter::MayaTransformWriter(const MDagPath & iDag, const pxrInternal_v0_8__pxrReserved__::SdfPath & uPath, bool instanceSource, pxrInternal_v0_8__pxrReserved__::usdWriteJobCtx & jobCtx) Line 502	C++
 	libusdMaya.dll!pxrInternal_v0_8__pxrReserved__::usdWriteJobCtx::_createPrimWriter(const MDagPath & curDag, bool instanceSource) Line 193	C++
 	libusdMaya.dll!pxrInternal_v0_8__pxrReserved__::usdWriteJobCtx::createPrimWriter(const MDagPath & curDag) Line 162	C++
 	libusdMaya.dll!pxrInternal_v0_8__pxrReserved__::usdWriteJob::beginJob(const std::basic_string<char,std::char_traits<char>,std::allocator<char> > & iFileName, bool append, double startTime, double endTime) Line 216	C++
 	libusdMaya.dll!pxrInternal_v0_8__pxrReserved__::usdTranslatorExport::writer(const MFileObject & file, const MString & optionsString, MPxFileTranslator::FileAccessMode mode) Line 197	C++

````

**Comment by sunyab** [comment]
This fix was merged into release 0.8.1, closing this out. Thanks!
